lib/svtplay_dl/__init__.py

```diff
@@ -442,7 +442,9 @@ def main():
     parser.add_option("--cmore-operatorlist", dest="cmoreoperatorlist", default=False, action="store_true",
                       help="show operatorlist for cmore")
     parser.add_option("--cmore-operator", dest="cmoreoperator", default=None, metavar="operator")
-                      
+    parser.add_option("--proxy", dest="proxy", default=None,
+                      metavar="proxy", help="access network through http/socks proxy")
```

Comment:
I think an example in the help text would be good, so you understand that you need the scheme for instance.

Here's how youtube-dl phrases it: https://github.com/rg3/youtube-dl/blob/master/README.md#network-options

The requests docs about proxies might also provide some inspiration: http://docs.python-requests.org/en/master/user/advanced/#proxies